docs/guide/api/apiblueprint.rst

```diff
@@ -1,18 +1,18 @@
 =============
-API Blueprint
+API blueprint
```

Comment:
This one should be "B" I think

Reply:
The exception would have to be API Blueprint, not Blueprint?

Reply:
Thanks for the careful review! I didn't even realize that "API Blueprint" was a proper noun. 

Blueprint was already an exception, but that file had other title case headings so I over corrected. 

Now I've fixed the heading, and updated the exception.